diff --git a/Formula/baymesh.rb b/Formula/baymesh.rb
index 36ba669..d543025 100644
--- a/Formula/baymesh.rb
+++ b/Formula/baymesh.rb
@@ -1,15 +1,14 @@
-# Documentation: https://docs.brew.sh/Formula-Cookbook
-#                https://rubydoc.brew.sh/Formula
-# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
 class Baymesh < Formula
   include Language::Python::Virtualenv
 
-  desc ""
-  homepage ""
+  desc "Command-line and library for members of Bayme.sh"
+  homepage "https://github.com/gtaylor/baymesh-cli"
   url "https://files.pythonhosted.org/packages/e7/19/fad6f32d430ac1dc44d6cb49cd4245fdb51b840c338e092489463ba50635/baymesh-0.3.3.tar.gz"
   sha256 "a279548dcb5dcceac4c4cf344fcf9fcc3bd32b1f90333543d4f25d5f707e6442"
-  license ""
+  license "GPL-3.0-only"
 
+  depends_on "rust" => :build
+  depends_on "libyaml"
   depends_on "python"
 
   resource "annotated-types" do
